packages/db/src/translate/file.ts

```diff
@@ -16,6 +17,20 @@ const getKey = (file: ApiFile) => {
   return undefined;
 };
 
+export const translatePostgresFileToApiFile = (
+  filePgRecord: PostgresFileRecord
+): Omit<ApiFile, 'granuleId'> => removeNilProperties({
+  bucket: filePgRecord.bucket,
+  checksum: filePgRecord.checksum_value,
+  checksumType: filePgRecord.checksum_type,
+  createdAt: filePgRecord.created_at.getTime(),
+  fileName: filePgRecord.file_name,
+  key: filePgRecord.key,
+  size: filePgRecord.file_size ? Number.parseInt(filePgRecord.file_size, 10) : undefined,
+  source: filePgRecord.source,
+  updatedAt: filePgRecord.updated_at?.getTime(),
+});
```

Comment:
I think this is what we want but definitely double-check me against the schema and type.

Reply:
I think it is, however I'm wondering if this should be translatePostgresFileToGranuleFile.... that might just be confusing though, as we're shortly not going to *have* an API file object that means anything.

Reply:
Hmm would we lose `ApiFile` after Dynamo is history? I would think we would still want the TypeScript type around because our API endpoint returns match this type (ish). I guess we'll have the api/models/schemas.js but I'm not totally sure what the plan there is.

I guess I don't love changing the name here right now because we're asserting an `ApiFile` type as the return. The function name matches that. If it was something like `translatePostgresFileToGranuleFile` I think it starts to get murky unless we want to change the return type.